examples/real-world/src/containers/UserPage.js

```diff
@@ -29,22 +22,17 @@ class UserPage extends Component {
     }
   }
 
-  handleLoadMoreClick() {
-    this.props.loadStarred(this.props.login, true)
-  }
+  handleLoadMoreClick = () => this.props.loadStarred(this.props.login, true)
 
-  renderRepo([ repo, owner ]) {
-    return (
-      <Repo repo={repo}
-            owner={owner}
-            key={repo.fullName} />
-    )
-  }
+  renderRepo = ([ repo, owner ]) => <Repo
```

Comment:
This shouldn’t be autobound (not an event handler).
This is also hard to read for somebody new to this syntax.
Please format it the way it was before.